### ISSUE TITLE
ES108FC1

### ISSUE THREAD

**Issue by vdp** [zephray/NekoInk#1]
May I ask on which reference is the pinout used for the ES108FC1 adapter based?

I see that there is for example GDSP signal on pin 43, which is not there in [the datasheet I've found](https://github.com/vroland/epdiy/files/10342125/ES108FC1_Ver0.1.pdf). I'm having trouble driving this screen- it just behaves strangely... For example it only updates at some random instant, then nothing happens, then it may or may not update some time later and so on... Having something left floating which shouldn't be would explain this erratic behaviour, I think. The above preliminary spec is the only 'official' document I've been able to find. This panel is used in Lenovo Yoga Book C930, but the only way to get a schematic seem to be through shady sites which ask to pay them. I would have considered it, but I'm not sure the doc they offer contains relevant info in the first place.

Thanks for your time!

**Comment by zephray** [comment]
Hi sorry forgot to reply last week.

It's mainly based on the datasheet you found. However I encountered the same issue as you did, the screen just doesn't work. From looking at the screen flex cable it's obvious these pins are certainly not NC. Basically from trial and error I noticed that connecting GDSP to pin 43 allowed it to work. This is later confirmed by looking at the schematics of C930 (one of my friend eventually decided to pull the trigger and purchased the schematics on one of those shady sites).

**Comment by vdp** [comment]
Thank you very much! This is __very__ helpful, because this screen is really driving me crazy. I see that there is a jumper for SDCE0 on pin 36- is it also required for the screen to work properly?

I will try to modify my adapter to see if I can make it work. Other than these 1 or 2 changes, is there anything else that I should watch for? For example I have a 10K pull-up R on the test pin 31, just as in your schematic. But I've noticed that sporadically all the lines are activated/charged at the same time, which is basically what pulsing the XON low does, as far as I can see from some driver chip datasheets I found online, so I wonder if a stronger(i.e. lower R value) pull-up may be needed... Also is there anything peculiar about the signal timing for example, that is different from how the other screens are typically driven?
